pkg/users/users.js

```diff
@@ -153,15 +161,33 @@ function AccountsPage() {
     } else if (path.length === 1) {
         return (
             <AccountDetails accounts={accountsInfo} groups={groupsExtraInfo}
-                            current_user={current_user_info?.name} user={path[0]} shells={shells} />
+                current_user={current_user_info?.name} user={path[0]} shells={shells} />
         );
     } else return null;
 }
 
-async function getLogins() {
+async function getLastlogPath() {
```

Comment:
This code should be shared with account-details.jsx, maybe by moving it into ./utils.js.

Reply:
will do